be/src/exec/pipeline/scan/scan_operator.cpp

```diff
@@ -411,6 +411,7 @@ Status ScanOperator::_trigger_next_scan(RuntimeState* state, int chunk_source_in
             SCOPED_THREAD_LOCAL_OPERATOR_MEM_TRACKER_SETTER(this);
 
             auto& chunk_source = _chunk_sources[chunk_source_index];
+            RETURN_IF_ERROR(chunk_source->start(state));
```

Comment:
Move this after L429. Otherwise, the eplased time of `start` is accounted to `io_task_wait_timer` not `io_task_exec_timer`.

Reply:
fixed